### PR TITLE
updated: home page with filters

### PR DESCRIPTION
**Pull request by rkapoor10**
- added filtered content to the home page
 - done through videosContainer using heading as props
 - here is how it looks 🤩
 -
ss

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
UI looks awesome

**Review by ankit9015 — APPROVED** [review]
Code is good to merge,

File: src/components/videosContainer/VideosContainer.jsx [hunk 1]
```diff
@@ -1,13 +1,23 @@
 import React from "react";
 import "./videosContainer.css";
 import VideoCard from "../cards/VideoCard";
+import { useVideo } from "../../context/VideoContext/VideoContext";
+
+const VideosContainer = ({ heading }) => {
+  const { videosData } = useVideo();
+  console.log(videosData);
```
Comment (t007rushi) [inline src/components/videosContainer/VideosContainer.jsx:8]: console here

**Review by t007rushi — APPROVED** [review]

File: src/components/videosContainer/VideosContainer.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/videosContainer/VideosContainer.jsx:8]: removed

**Pull request merged**